### ISSUE TITLE
Share on other social medias

### ISSUE THREAD

**Issue by BaptisteGelez** [Plume-org/Plume#15]
I'm thinking of Facebook, Twitter, Mastodon and Diaspora (and by email?) at least, but there could be others to be included.

**Comment by dfeyer** [comment]
https://github.com/heiseonline/shariff can be a source of inspiration, with respect to use privacy.

**Comment by BaptisteGelez** [comment]
Yes, when implementing this, official widgets with trackers should be avoided.

**Comment by BaptisteGelez** [comment]
I wonder if this really makes sense…
